extensions/emmet/src/util.ts

```diff
@@ -227,10 +227,14 @@ export function parsePartialStylesheet(document: vscode.TextDocument, position:
 
 	function consumeBlockCommentBackwards() {
 		if (stream.peek() === slash) {
-			if (stream.backUp(1) === star) {
-				stream.pos = findOpeningCommentBeforePosition(stream.pos) ?? startOffset;
-			} else {
-				stream.next();
+			switch (stream.backUp(1)) {
+				case star:
+					stream.pos = findOpeningCommentBeforePosition(stream.pos) ?? startOffset;
+					break;
+				case slash:
```

Comment:
I wonder if this might affect `//` comments?
Also, if the issue only occurs when we're at offset 0, then I'm thinking we can check for that instead.
There's `document.offsetAt` and `document.positionAt` to convert back and forth.

Reply:
I don't believe `//` comments should be effected, as the same loop checks the entire line for the presence of `//`. Yes, some lines that only have one character on them may be skipped, but it isn't possible to have `//` on a line with one character.

I initially approached the resolution this way as I thought it was possible to trigger the issue with positions that aren't offset 0, but I now believe it isn't possible. You're right, checking that we're currently at offset 0 is the better fix. I'll modify this PR to check for start of file.